src/stylesheets/components/_card.scss

```diff
@@ -25,22 +32,20 @@
   .sds-card__header-actions {
     display: flex;
   }
-
 }
 
 .sds-card .sds-card__content {
   padding: 15px 0 0;
   display: block;
-
   @media only screen and (max-width: 600px) {
     display: none;
   }
 }
-
 //search fieldset
```

Comment:
The search component is common and reusable, we should move these styles out to a separate file.